rust/avdschema/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "avdschema"
+version = "0.0.0"
```

Comment:
different version than in pyproject.toml (I know this is temporary but should we keep them in sync?)